media-art-background.js

```diff
@@ -17,23 +46,27 @@ function setBackground(root, lovelace, app, originalStyle) {
     const backgroundUrl = entityInfo.attributes[entityImageAttribute];
     if (!backgroundUrl) continue;
 
-    app.style.backgroundImage = `url('${backgroundUrl}')`
-    app.style.backgroundRepeat = 'no-repeat';
-    app.style.backgroundPosition = 'center';
-    app.style.backgroundSize = 'cover';
-    app.style.backgroundAttachment = 'fixed';
+    bgroundElem.style.backgroundImage = `url('${backgroundUrl}')`
+    bgroundElem.style.opacity = 1;
 
-    return;
+    return; // abort after first element with valid background
   }
 
-  app.style = originalStyle;
+  setupStyle(lovelace, bgroundElem);
```

Comment:
I'm pretty sure this no longer unsets the media background now

Reply:
jes but by setting opacity to 0 the image will be hidden and if the new transition parameter is configured, it will also fade out or in the background

Reply:
so basically the image will still be there but it is transparent

Reply:
Oh, I totally didn't notice that while reading the code, fancy! Sadly whenever the entity picture changes it is still instant, but I totally love the rest of this. Kinda feel that a 10px blur and 5s transition should be the default after playing with your version a bit

Reply:
yes maybe i will add a crossfade between backgrounds in the soon :)
and i agree that for most users different defaults would be better to really have a nice background

Reply:
Well, if you want, I can already merge this because it's definitely an improvement? :)

Reply:
yes you could :)
i will add additional features in a separate pr